### ISSUE TITLE
Split handedness at runtime / boottime?

### ISSUE THREAD

**Issue by elrom** [zmkfirmware/zmk#393]
qmk has a pretty nifty feature. It allows handedness of split keyboards to be determined at run (boot?) time, by reading a pin / matrix_pin / eeprom.

I'm wondering if something similar can be done in zmk? Or is too much of the logic already done at compile time?

**Comment by nmunnich** [comment]
I don't believe that this is something that will ever be done in ZMK. It would require significant reimagination with little gain, as simply flashing left and right halves is perfectly fine.
